### DISCUSSION TITLE
Planned release date for Python 3.11.2

### DISCUSSION THREAD

**Discussion by rafrafek** [pyenv/pyenv#2621] [General]
Hi! I can see commit adding Python 3.11.2 is merged to master, but it is not released yet. What is the planned release date for the next version of pyenv including it? Thanks!

**Comment by anton-petrov** [comment]
Released https://github.com/pyenv/pyenv/releases/tag/v2.3.13

**Reply by rafrafek** [reply]
Thanks!
